integration/setup/config.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 const (
-	namespace    = "giantswarm"
-	organization = "giantswarm"
-	quayAddress  = "https://quay.io"
+	namespace       = "giantswarm"
+	organization    = "giantswarm"
+	tillerNamespace = "kube-system"
```

Comment:
This is where the tiller is installed in e2e setup.

Reply:
Isn't this the default?

Reply:
Yes it's the default tiller namespace.